Graphics/GraphicsEngine/interface/DeviceContext.h

```diff
@@ -910,6 +910,35 @@ static const Uint32 TLAS_INSTANCE_OFFSET_AUTO = ~0u;
 /// AZ TODO
 static const Uint32 TLAS_INSTANCE_DATA_SIZE = 64;
 
+/// AZ TODO
+struct InstanceMatrix
+{
+    ///  rotation    translation
+    /// (0  1  2)   [ 3]
+    /// (4  5  6)   [ 7]
+    /// (8  9 10)   [11]
+    float data [3][4];
```

Comment:
[4][3]? At least this is what depicted in the comment

Reply:
will be replaced by
```
    /// (0.0 1.0 2.0)
    /// (0.1 1.1 2.1) - rotation
    /// (0.2 1.2 2.2)
    /// 
    /// [0.3 1.3 2.3] - translation
```